corehq/apps/export/templates/export/bootstrap3/customize_export.html

```diff
@@ -0,0 +1,184 @@
+{% extends "style/bootstrap3/base_section.html" %}
+{% load hq_shared_tags %}
+{% load i18n %}
+
+{% block js %}
+    {{ block.super }}
+    <script src="{% static 'style/lib/knockout_plugins/knockout_mapping.ko.min.js' %}"></script>
+    <script src="{% static 'style/ko/knockout_bindings.ko.js' %}"></script>
+    <script src="{% static 'export/js/exports.js' %}"></script>
+    <script src="{% static 'export/js/const.js' %}"></script>
+    <script src="{% static 'export/js/models.js' %}"></script>
+    <script src="{% static 'export/js/utils.js' %}"></script>
+{% endblock %}
+
+{% block js-inline %}{{ block.super }}
+    <script>
+        $(function () {
+            var customExportView = new Exports.ViewModels.ExportInstance(
+                {{ export_instance|JSON }},
+                {
+                    saveUrl: {{ request.get_full_path|safe|JSON }}
+                }
+            );
+
+            $('#customize-export').koApplyBindings(customExportView);
+            $('.export-tooltip').tooltip();
+        });
+    </script>
+{% endblock %}
+
+{% block page_content %}
+<div id="customize-export" style="display: none;" data-bind="visible: true">
+    {# content for multi-select help popover #}
+    {% if export_instance.type == 'case' %}
+    <div id="popover_content_wrapper" style="display: none">
+        {% blocktrans %}
+            These options allow you to configure how CommCare exports data from multi-select questions.
+            If you do not want to split the options into multiple columns select the 'plain' option.
+        {% endblocktrans %}
+        <br/>
+        {% blocktrans %}
+            Each item that is selected in the select list will appear as a column in the exported data.
+            Any options not selected will be in an "extra" column.
+        {% endblocktrans %}
+        <a href='https://help.commcarehq.org/display/commcarepublic/Splitting+multi-select+data+in+exports' target="_blank">
+            {% trans "More info" %}
+        </a>
+    </div>
+    {% endif %}
+    {% include "export/partial/customize_export_header.html" %}
+    <form class="form-horizontal" method="post">
+        {% csrf_token %}
+        <fieldset>
+            <div class="form-group">
+                <label
+                    for="export-name"
+                    class="col-sm-2 control-label">
+                    {% trans "Export Name" %}
+                </label>
+                <div class="col-sm-4">
+                    <input type="text" class="form-control" id="export-name" data-bind="value: name" />
+                </div>
+            </div>
+            <div class="form-group">
+                <label
+                    for="format-select"
+                    class="col-sm-2 control-label">
```

Comment:
`col-sm-2` seems really small for actual `sm` screens and a lot of these label sizes

recommend: 
for labels `col-xs-4 col-md-3 col-lg-2`
for fields `col-xs-8 col-md-8 col-lg-9`